plugins/tours/tests/phpunit/unit/renderTourComments.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Tests for render_tour_comments().
+ *
+ * @since       1.0.0
+ * @author      Robert Gadon <rgadon107>
+ * @package     spiralWebDb\CornerstoneTours\Tests\Unit
+ * @link        https://github.com/rgadon107/cornerstone
+ * @license     GNU-2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Tests\Unit;
+
+use Brain\Monkey\Functions;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+use function spiralWebDb\CornerstoneTours\render_tour_comments;
+
+/**
+ * @covers ::\spiralWebDb\CornerstoneTours\render_tour_comments
+ *
+ * @group   tours
+ */
+class Tests_RenderTourComments extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once TOURS_ROOT_DIR . '/src/meta.php';
+	}
+
+	/**
+	 * @dataProvider addTestData
+	 */
+	public function test_should_echo_meta_key_values_when_postmeta_exists( $tour_id, $postmeta ) {
+		Functions\expect( 'get_post_meta' )
+			->once()
+			->with( $tour_id, 'tour_comments', true )
+			->andReturn( $postmeta );
+
+		$this->expectOutputString( $tour_id );
```

Comment:
This is not what's expected. The tour ID isn't returned, but rather escaped tour comments' value or HTML. Look at the source code:

https://github.com/rgadon107/cornerstone/blob/b12d9c8aa0593a6a4a6d7af418e2a78c8234a1b3/plugins/tours/src/meta.php#L38-L46

Let's build a unit test that matches the test data that's given to it. How?

```php
public function test_should_echo_meta_key_values_when_postmeta_exists( $tour_id, $meta, $expected ) {
	Functions\expect( 'get_post_meta' )
		->once()
		->with( $tour_id, 'tour_comments', true )
		->andReturn( $meta );

	if ( ! is_null( $expected ) {
		$this->expectOutputString( $expected );
	}

	render_tour_comments( $tour_id );
}

You'll need to add the expected value or HTML in the test data and update the scenarios.